release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -515,6 +515,15 @@ Enabling the `TechPreviewNoUpgrade` feature set on your cluster cannot be undone
 
 For more information, see xref:../nodes/pods/nodes-pods-configuring.adoc#pod-disruption-eviction-policy_nodes-pods-configuring[Specifying the eviction policy for unhealthy pods].
 
+[id="ocp-4-13-graceful-node-shutdown"]
+==== Support for graceful node shutdown
+
+A graceful node shutdown delays the eviction of pods during a node shutdown. In {product-title} {product-version}, you can configure the kubelet to enable a graceful node shutdown to ensure that pods running critical workloads are not interrupted.
+
+To configure graceful node shutdowns, specify a termination grace period for regular and critical pods in the `KubeletConfig` custom resource. A termination grace period defines a time period for the pod to complete any ongoing tasks before terminating. You can also add priority classes to pods to specify the order of termination.
+
+For further information see xref:../nodes/nodes/nodes-nodes-graceful-shutdown.adoc#nodes-nodes-graceful-shutdown[Managing graceful node shutdown]
```

Comment:
If this gets merged before the main docs, I think we might should comment this line out for now just in case.

Reply:
I see your checks failed. This link would be the cause.